app/experimenter/visualization/api/v1/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from experimenter.visualization.api.v1.views import analysis_results_view
+
+urlpatterns = [
+    url(
+        r"^(?P<slug>[\w-]+)/$",
```

Comment:
Move the `visualizations/` prefix out of the project wide urls.py and into this line